scenario_editor/package.json

```diff
@@ -6,6 +6,7 @@
     "node": ">=14.4.0"
   },
   "dependencies": {
```

Comment:
you never actually add the data sheet as a dependency. Run `npm i --save react-datasheet`